code/turtlesim/package.xml

```diff
@@ -1,21 +1,18 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>turtlesim</name>
+  <name>turtlesim_square_move</name>
```

Comment:
Having a package name different than the project name raises the following error:
```
CMake Error at /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_package_xml.cmake:53 (message):
  ament_package_xml() package name 'turtlesim_square_move' in '/package.xml'
  does not match current PROJECT_NAME 'docs_turtlesim'.  You must call
  project() with the same package name before.
Call Stack (most recent call first):
  /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:226 (ament_package_xml)
  CMakeLists.txt:16 (rosidl_generate_interfaces)
```